NLog.Targets.Sentry.UnitTests/SentryTargetTests.cs

```diff
@@ -40,14 +56,6 @@ public void TestPublicConstructor()
             });
         }
 
-        [Test]
```

Comment:
Wouldn't we still want a unit test like this to make sure that bad DSN fails correctly?

Reply:
Why should we test the functionality of the SharpRaven library? We need to test only our code.

Reply:
Right. We need to take care of the correct DSN in advance.

Reply:
> We need to take care of the correct DSN in advance.

What do you mean by this?  Do you mean that developers should fix the DSN they provide or that the Sentry Target should handle bad DSNs?

Reply:
I mean that developers should provide the correct DSN.

Reply:
I do agree with that.  Are we fine with the first log being the place where the developers find out if the DSN is valid or not?  Because with the client being lazy, the DSN isn't used until the first time something is logged.

Reply:
Right. I think this is not a good idea. It will be great when the Sentry Target is initialized. Perhaps I will fix this in the future.